NEWS

```diff
@@ -0,0 +1,5 @@
+version 0.0.3
+- fixed LaTeX problem caused by double loading of \usepackage{color} with different
+  options. Occurred when specifying 'letterhead' while 'colorlinks' was 'false'.
+
```

Comment:
We have an existing file `inst/NEWS.Rd` -- can you add there instead please?

Reply:
Done in https://github.com/eddelbuettel/linl/pull/18/commits/a833f4abdb17c85064fadb8e6344855aca039c43